content/en/docs/ops/ambient/usage/ztunnel/index.md

```diff
@@ -0,0 +1,545 @@
+---
+title: Ztunnel Networking - L4 & M-TLS
+description: User guide for ztunnel networking.
+weight: 2
+owner: istio/wg-networking-maintainers
+test: n/a
+---
+
+## Introduction 
+
+This guide describes the functionality and usage of the Ztunnel proxy and Layer-4 networking functions using Istio Ambient mesh. We use a sample user journey to describe these functions. 
+
+The Ztunnel (Zero Trust Tunnel) component is a purpose-built per-node proxy for Istio ambient mesh. It is responsible for securely connecting and authenticating workloads within ambient mesh. Ztunnel is designed to focus on a small set of features for your workloads in ambient mesh such as mTLS, authentication, L4 authorization and telemetry, without terminating workload HTTP traffic or parsing workload HTTP headers. The ztunnel ensures traffic is efficiently and securely transported to the waypoint proxies, where the full suite of Istio’s functionality, such as HTTP telemetry and load balancing, is implemented.  Sometimes the term "Secure Overlay Networking" is also used informally to collectively describe the set of L4 networking functions implemented in an Ambient mesh via the Ztunnel proxy.
+
+It is expected that some production use cases of Istio in Ambient mode may be addressed solely via the L4 Secure overlay networking features whereas other use cases will additionally need advanced traffic management and L7 networking features for which the additional Waypoint proxies will need to be deployed. This is summarized in the following table. This guide focusses on functionality related to the baseline L4 and mTLS networking using ztunnel proxies. Other guides cover the advanced L7 networking functions that additionally require waypoint proxies. 
```

Comment:
would it be possible not mention L7 other than a simple link to keep this focus on L4?  The reason is user may not be interested in L7 at all.

Reply:
Yes, the guide is for L4. L7 is only mentioned where absolutely necessary to explain the L4 function or how ztunnel separates out the L4 from the L7. Same logic was used in other ztunnel L4 blog posts also.